e2e/tests/workspace/workspace-delete.spec.ts

```diff
@@ -0,0 +1,70 @@
+import WorkspacesPage from 'app/page/workspaces-page';
+import {findWorkspace, signIn} from 'utils/test-utils';
+import WorkspaceCard from 'app/component/workspace-card';
+import {EllipsisMenuAction, WorkspaceAccessLevel, LinkText} from 'app/text-labels';
+import Modal from 'app/component/modal';
+
+
+describe('Delete workspace', () => {
+
+  beforeEach(async () => {
+    await signIn(page);
+  });
+
+  /**
+   * Test:
+   * - Find an existing workspace. Create a new workspace if none exists.
+   * - Select "Delete" thru the Ellipsis menu located on the Workspace card.
+   * - Get the content on the modal
+   * - Click the delete workspace button on the modal
+   * - Navigate to the "View Workspaces" page
+   * - Verify that the workspace has been delete
+   */
+  
+  describe('From "Your Workspaces" page using Workspace card ellipsis menu', () => {
+
+    test('As OWNER, user can delete workspace', async () => {
+      const workspacesPage = new WorkspacesPage(page);
+      const workspaceCard = await findWorkspace(page);
+      const workspaceName = await workspaceCard.getWorkspaceName();
+      console.log(workspaceName);
+
+      // Verify Workspace Access Level is OWNER.
+      const accessLevel = await workspaceCard.getWorkspaceAccessLevel();
+      expect(accessLevel).toBe(WorkspaceAccessLevel.Owner);
+      
+      await workspaceCard.asElementHandle().hover();
+      // click on Ellipsis "Delete"
+      await (workspaceCard.getEllipsis()).clickAction(EllipsisMenuAction.Delete, { waitForNav: false });
+
+      const modal = new Modal(page);
```

Comment:
Can you create a reusable function for delete workspace modal similar to this one? It's okay if you don't feel comfortable doing it as this is your first PR.
```
/**
   * Click DELETE COHORT button in Cohort Delete Confirmation dialog.
   * @return {string} Dialog textContent.
   */
  async deleteConfirmationDialog(): Promise<string> {
    const modal = new Modal(this.page);
    const contentText = await modal.getContent();
    await modal.clickButton(LinkText.DeleteCohort, {waitForClose: true});
    await waitWhileLoading(this.page);
    return contentText;
  }
```

Reply:
working on it!

Reply:
Done!